File: Tests/Variables/Vars.js
let score = 0;

// Block scope
{
    let coins = 5;
    let bonus = 10;

    let total = coins * bonus;
    score += total;

    console.log("Inside block:", total);
}

// Outside block
console.log("Final score:", score);
console.log("Coins outside block:", coins); // Would error: coins not defined

File: Tests/Variables/Vars.py
score = 0

# Block scope (Python has function-level scope)
def block():
    coins = 5
    bonus = 10

    total = coins * bonus
    global score
    score += total

    print("Inside block:", total)

block()

# Outside block
print("Final score:", score)
print("Coins outside block:", coins) # Would error: coins undefined